Dockerfile_mbtci_java17

```diff
@@ -1,93 +1,189 @@
 
 # Environment variables
 ENV PYTHON /usr/bin/python3
-ENV M2_HOME=/opt/maven/apache-maven-${MAVEN_VERSION}
+ENV MAVEN_HOME /usr/share/maven
+ENV M2_HOME ${MAVEN_HOME}
 ENV GOPATH /go
 ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 ENV CGO_ENABLED=0
 ENV GOOS=linux
 ENV DEBIAN_FRONTEND=noninteractive
 
+# create user account and using openssl to create password
+RUN set -ex \
+  && apt-get update \
+  && apt-get install -y openssl --no-install-recommends \
+  && rm -rf /var/lib/apt/lists/* \
+   # smoke test
+  && openssl version \
+  && useradd --home-dir ${MTA_USER_HOME} \
+            --create-home \
+            --shell /bin/bash \
+            --user-group \
+            --uid 1000 \
+            --comment 'Cloud MTA Build Tool' \
+            --password "$(echo weUseMta | openssl passwd -1 -stdin)" ${MTA_USER} \
+  # allow anybody to write into the image user home directory
+  && chmod a+w ${MTA_USER_HOME}
+
 ADD http://aia.pki.co.sap.com/aia/SAP%20Global%20Root%20CA.crt \
     /etc/ssl/certs/SAP_Global_Root_CA.crt
 
-# Download required env tools
-RUN apt-get update && \
-    apt-get install --yes --no-install-recommends \
-		ca-certificates \
-		git \
-		curl && \
-
-    # Change security level as the SAP npm repo doesnt support buster new security upgrade
-    # the default configuration for OpenSSL in Buster explicitly requires using more secure ciphers and protocols,
-    # and the server running at http://npm.sap.com/ is running software configured to only provide insecure, older ciphers.
-    # This causes SSL connections using OpenSSL from a Buster based installation to fail
-    # Should be remove once SAP npm repo will patch the security level
-    # see - https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=912759
-    sed -i -E 's/(CipherString\s*=\s*DEFAULT@SECLEVEL=)2/\11/' /etc/ssl/openssl.cnf && \
-
 # install node
-	NODE_HOME=/opt/nodejs; mkdir -p ${NODE_HOME} && \
-    curl --fail --silent --output - "https://nodejs.org/dist/${NODE_VERSION}/node-${NODE_VERSION}-linux-x64.tar.gz" \
-     | tar -xzv -f - -C "${NODE_HOME}" && \
-    ln -s "${NODE_HOME}/node-${NODE_VERSION}-linux-x64/bin/node" /usr/local/bin/node && \
-    ln -s "${NODE_HOME}/node-${NODE_VERSION}-linux-x64/bin/npm" /usr/local/bin/npm && \
-    ln -s "${NODE_HOME}/node-${NODE_VERSION}-linux-x64/bin/npx" /usr/local/bin/ && \
-    npm install --prefix /usr/local/ -g grunt-cli && \
-	
-# install ui5-cli temporay solution
-	npm install --prefix /usr/local/ -g @ui5/cli && \
+RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
+    && case "${dpkgArch##*-}" in \
+      amd64) ARCH='x64';; \
+      ppc64el) ARCH='ppc64le';; \
+      s390x) ARCH='s390x';; \
+      arm64) ARCH='arm64';; \
+      armhf) ARCH='armv7l';; \
+      i386) ARCH='x86';; \
+      *) echo "unsupported architecture"; exit 1 ;; \
+    esac \
+    && set -ex \
+    && apt-get update \
+    && apt-get install -y ca-certificates curl gnupg dirmngr xz-utils libatomic1 --no-install-recommends \
```

Comment:
Get the comment for libatomic1 requirement

Reply:
OK